examples/communities/louvain.py

```diff
@@ -105,7 +133,7 @@ def delta_modularity(self, u: int, alpha: int, *, exact: bool = False) -> float:
         Args:
             u: Integer label of the nx.Node to be moved
             alpha: Integer label of the target community
-            exact (bool, optional): If true, calculate two modularities and calculate delta, otherwise use approximative formula. Defaults to False.
+            exact (bool, optional): If true, calculate two full graph modularities and subtract them to receive delta, otherwise use one-move delta-formula. Defaults to False.
```

Comment:
Is there some case where `exact=True` and `exact=False` differ? 
If yes, the cases should be documented.
If no, then only the faster option should be retained.

Reply:
there should be no more difference. Removed the slow option.